webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdatePlan.scala

```diff
@@ -71,23 +71,23 @@ object RepositoryUpdatePlan {
    */
   val builtInNamedGraphs: Set[RdfDataObject] = Set(
     RdfDataObject(
-      path = "knora-ontologies/knora-admin.ttl",
+      path = "webapi/src/main/resources/knora-ontologies/knora-admin.ttl",
       name = "http://www.knora.org/ontology/knora-admin"
     ),
     RdfDataObject(
-      path = "knora-ontologies/knora-base.ttl",
+      path = "webapi/src/main/resources/knora-ontologies/knora-base.ttl",
       name = "http://www.knora.org/ontology/knora-base"
     ),
     RdfDataObject(
-      path = "knora-ontologies/salsah-gui.ttl",
+      path = "webapi/src/main/resources/knora-ontologies/salsah-gui.ttl",
       name = "http://www.knora.org/ontology/salsah-gui"
     ),
     RdfDataObject(
-      path = "knora-ontologies/standoff-onto.ttl",
+      path = "webapi/src/main/resources/knora-ontologies/standoff-onto.ttl",
       name = "http://www.knora.org/ontology/standoff"
     ),
     RdfDataObject(
-      path = "knora-ontologies/standoff-data.ttl",
+      path = "webapi/src/main/resources/knora-ontologies/standoff-data.ttl",
```

Comment:
Please dont update here, this is loading from the class path

Reply:
✔️